gems/pending/appliance_console/locales/container/en.yml

```diff
@@ -9,6 +9,7 @@ en:
     - db_config
     - db_replication
     - db_maintenance
+    - log_config
```

Comment:
@carbonin We have no access or control of underlying hardware on containers as storage is presented by docker, openshift or k8s. If an additional persistent volume was going to be added for logging it would have to be handled by the provider itself and not directly initiated from the appliance.